README.md

```diff
@@ -101,6 +101,47 @@ output.render()
 
 \[TODO @IvanPombo: add gif with render\]
 
+### Fluid tank
+
+This scenario simulates the motion of a fluid in a cubic or cylindrical tank.
+Fluid is injected in the tank via an inlet located at the top of the tank, and
+flows out of the tank via an outlet located at the bottom of the tank. The
+motion of the fluid is controled by gravity. The simulation is performed using
+the [Smoothed Particle Hydrodynamics](https://en.wikipedia.org/wiki/Smoothed-particle_hydrodynamics)
+method.
+
+#### Example
+
+Initialize the scenario:
+
+```python
+from inductiva import fluids
+scenario = fluids.scenarios.FluidTank(fluid=fluids.WATER, fluid_level=0.5)
+```
+
+The user can specify the fluid (e.g. water, honey or oil), the fluid level (in
+meters), as well as the tank shape and its inlet and outlet properties.
+
+Run the simulation:
+
+```python
+output = scenario.simulate(simulation_time=5,
```

Comment:
I think we should specify that this is only available for SPlisHSPlasH at the moment, since other SPH scenarios contain both.